.vscode/launch.json

```diff
@@ -22,7 +22,18 @@
             "request": "launch",
             "name": "Debug Frontend: Launch Chrome against http://localhost:5173",
             "url": "http://localhost:5173",
-            "webRoot": "${workspaceFolder}/frontend"
+            "webRoot": "${workspaceFolder}/frontend",
+            // "runtimeExecutable": "/usr/bin/google-chrome"
+        },
+    ],
+    "compounds": [
+        {
+            "name": "Debug Fullstack",
```

Comment:
I am pretty sure this should work, but was not able to test it yet, due to #729.